app/src/pages/AdminPage.tsx

```diff
@@ -0,0 +1,9 @@
+import { User } from "../components/Types";
+
+export const checkAdmin = (user: User | null) => {
```

Comment:
I like it when booleans, or functions that return booleans, start with `is`, `has`, `should`, etc.